mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -1174,6 +1316,19 @@ def coupled_ns_heat_operator(
 
         The tuple `(fluid_rhs, wall_rhs)`.
     """
+    from warnings import warn
+    warn(
+        "coupled_ns_heat_operator is deprecated and will disappear in Q3 2023. "
+        "Set up interface boundaries explicitly via"
+        ":func:`add_interface_boundaries` and include them when calling the "
+        "individual operators instead.", DeprecationWarning, stacklevel=2)
+
+    if not use_kappa_weighted_grad_flux_in_fluid:
```

Comment:
Why don't we remove it right now and make the warning says that it is being ignored?

Reply:
Done in 6cd4494.